src/main/java/fi/aalto/cs/apluscourses/intellij/actions/ImportModuleAction.java

```diff
@@ -0,0 +1,66 @@
+package fi.aalto.cs.apluscourses.intellij.actions;
+
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import fi.aalto.cs.apluscourses.intellij.services.MainViewModelProvider;
+import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
+import fi.aalto.cs.apluscourses.model.Course;
+import fi.aalto.cs.apluscourses.model.Module;
+import fi.aalto.cs.apluscourses.model.ModuleInstaller;
+import fi.aalto.cs.apluscourses.model.ModuleInstallerImpl;
+import fi.aalto.cs.apluscourses.presentation.CourseViewModel;
+import fi.aalto.cs.apluscourses.presentation.base.BaseViewModel;
+import fi.aalto.cs.apluscourses.utils.async.SimpleAsyncTaskManager;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+
+public class ImportModuleAction extends AnAction {
+
+  public static final String ACTION_ID =
```

Comment:
```java
public static final String ACTION_ID = ImportModuleAction.class.getCanonicalName();
```
produces exactly the same result ("fi.aalto.cs.apluscourses.intellij.actions.ImportModuleAction"), could this work?

Reply:
I wasn't aware of that. A very good idea! I will change this.